### DISCUSSION TITLE
Ignoring template strings when formatting on save

### DISCUSSION THREAD

**Discussion by zakissimo** [denoland/deno#13802] [Q&A]
I use deno to format my javascript on save. Sadly my template strings are put on multiple lines or in one line according to the formatting rules but it sometimes goes against my desired output (since the format of template strings is translated in the output string).

Formatted by deno:
![image](https://user-images.githubusercontent.com/64891486/156148068-c1cd9ea2-7d46-4d53-9033-735b6e290489.png)

Is there a way for me to exclude template strings from being processed by deno ? Maybe everything between backticks ?

Desired:
![image](https://user-images.githubusercontent.com/64891486/156147820-085e27cc-50f5-4bc3-a2e2-1ab292550953.png)

I'm using deno with null-ls in neovim. I can provide you with any additional info you might need.

Thanks

**Comment by KhaledGabr** [comment]
Have you figured a solution for this. It's very annoying indeed.

**Reply by Seally** [reply]
Adding `// deno-fmt-ignore` on the line before the `return` should work.
https://deno.land/manual@v1.26.1/tools/formatter#ignoring-code
